### DISCUSSION TITLE
It flashes when i open the cover to read

### DISCUSSION THREAD

**Discussion by ating86** [koreader/koreader#12735] [Q&A] [answered]
When I open Koreader, turn off the backlight, and open the protective cover to read, it will flash. 
I found that the backlight cannot be turned off in the built-in Kindle reader. Is the reason for this flashing due to the Kindle firmware? 
My device is Kindlepaperwhite3. The Koreader version is v2024.04. The Kindle firmware is 5.10.3

**Comment by NiLuJe** [comment]
Define "flash".

> due to the Kindle firmware?

Probably, we don't handle power management ourselves on Kindle.

**Reply by ating86** [reply]
When I set KOReader to turn off the backlight, the screen briefly lights up when I open the cover, but then the backlight turns off again immediately.

**Reply by NiLuJe** [reply, marked as answer]
Yeah, we can't avoid that on Kindle ;).
